watertap/examples/flowsheets/case_studies/groundwater_treatment/groundwater_treatment.py

```diff
@@ -199,124 +348,111 @@ def add_costing(m):
 
     m.fs.costing.add_electricity_intensity(m.fs.feed.properties[0].flow_vol)
 
-    m.fs.costing.spent_media_recovery_mass = Expression(
+    m.fs.costing.annual_water_inlet = Expression(
         expr=m.fs.costing.utilization_factor
         * pyunits.convert(
-            m.fs.byproduct.flow_mass_comp[0, "filtration_media"],
-            to_units=pyunits.kg / m.fs.costing.base_period,
-        ),
-        doc="Mass of spent filtration media generated per year (accounting for utilization factor)",
+            m.fs.feed.properties[0].flow_vol,
+            to_units=pyunits.m**3 / m.fs.costing.base_period,
+        )
     )
 
-    m.fs.costing.cost_spent_media_disposal = Expression(
-        expr=pyunits.convert(
-            m.fs.costing.spent_media_recovery_mass
-            * m.fs.costing.filtration_media_disposal_cost,
-            to_units=m.fs.costing.base_currency / m.fs.costing.base_period,
-        ),
-        doc="Disposal cost of spent filtration media per year (accounting for utilization factor)",
+    m.fs.costing.add_LCOW(m.fs.filtered_water.properties[0].flow_vol)
```

Comment:
Although the LCOW with respect to the product water flow rate will be very similar in this case, our standard go-to levelized cost metric should be the levelized cost with respect to influent flowrate.

Reply:
Are you sure that influent flowrate is supposed to be our default basis for LCOW? I based a lot of these changes on what I saw in Tim's metab example, and in that flowsheet, product water is used as the basis for the LCOW calculation.

Reply:
We've been using levelized cost of treatment (LCOT) to denote levelized cost per influent flow rate. I am sure that if we want a metric that is broadly applicable across all projects, levelized cost per influent flowrate will be the one of interest, at least for wastewater treatment applications.

Additionally, what we name the metric (i.e., LCOT or LCOW_feed) is fairly subjective. The important point here is that levelized cost per feed flowrate should always be relevant, and in some cases, levelized cost per product water flowrate may also be relevant. 

For this application, both metrics happen to be relevant.

Reply:
Also note- if we are looking to the METAB flowsheet as the example- both LCOW and LCOT are expressions on the flowsheet (not to mention other levelized cost metrics relevant to that particular use case). 

And for the sake of clarity and being explicit while reporting, we can always say something like LCOW with respect to product water flowrate, LCOW with respect to influent flowrate. Alternatively we can just have LCOW and LCOT and be explicit about LCOT representing division by influent flow rate.

Reply:
On my latest commit, I changed the LCOW metric to use feed flowrate as the basis. As you mentioned, it shouldn't really have any effect, because for this flowsheet, feed flowrate and outlet flowrate are essentially identical. But now it is following the preferred convention.